### DISCUSSION TITLE
Player Indicator with Combat Level Next to Players Name

### DISCUSSION THREAD

**Discussion by Hehemaxwell** [runelite/runelite#16703] [Q&A]
Does anyone know if there is a plugin/an easy way to add to the player indicator plugin that will show the players combat level next to their name? I have been looking around and haven't found anything on the plugin hub. Can anyone help point me in the right direction?
See attached image for what I am looking for.

Thank you!

![image](https://github.com/runelite/runelite/assets/124599994/a0e4b23b-db9d-4b89-9419-a7fdf089cb51)

**Comment by Nightfirecat** [comment]
We have declined this multiple times in the past in a variety of forms, because it violates the following item [from Jagex's Third Party Client Guidelines statement](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1):

> Offers additional information about other players, with the purpose of scouting PvP targets

**Reply by Hehemaxwell** [reply]
Thank you for letting me know!
